Chapter7/call_count.erl

```diff
@@ -0,0 +1,53 @@
+-module(call_count).
+-export([count/2]).
+-export([counter/1]).
+
+
+% Exercise 7-7: Counting Calls
+% How can you use the Erlang macro facility to count the number of calls to a particular function in a particular module?
+
+% No idea how to go about doing this? Does it want me to give it a file name of a file with erlang code
```

Comment:
yeah, book doesn't say clearly what it means, though I'd assume, since this chapter related to macroses, you need to use some [predefined macroses](http://erlang.org/doc/reference_manual/macros.html) to solve this problem.

Confusing part here is to where to store that counter. As an option you can spawn a process that receives a messages, once function is called and store it in a state. Your macro then should post this messages with function name and module name to that process.
Another option is to put this info into process dictionary (using `put/2` and `get/1` functions).

use case example:
```erlang
foo(Bar) ->
    ?COUNT_IT,
    % ...
```
or
```erlang
?count_call(module_name, foo, [Bar]).
```